src/dbspro/cli/run.py

```diff
@@ -23,7 +23,7 @@ def main(args):
     # Check if directory exists. If not make one.
     if not os.path.isdir(args.directory):
         os.mkdir(args.directory)
-        logging.info(f'Output directory {args.directory} created.')
+        logging.info(f"Output directory {args.directory} created.")
 
     # Append full path to targets.
     targets_with_path = [f"{args.directory}/{t}" for t in args.targets] if args.targets else None
```

Comment:
```suggestion
```
This an be removed and `args.targets` used directly.

Reply:
I think removing it will demand the user to be in the directory, but maybe that's ok?

Reply:
Yeah I think it is already required to be in the directory, this is how the config and ABC-sequences are found. So we should remove this and also the  `args.directory` option.

Reply:
Implementing it.

Reply:
I tested it and it seems like snakemake is smart about this if we pass it into the workdir=args.dir to the snakemake command (so it works running it remotely after removing it as well).